var/spack/repos/builtin/packages/care/package.py

```diff
@@ -50,137 +67,219 @@ class Care(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("cxx", type="build")  # generated
     depends_on("fortran", type="build")  # generated
 
-    variant("openmp", default=False, description="Build Shared Libs")
+    variant("openmp", default=False, description="Build with OpenMP support")
+    variant("mpi", default=False, description="Enable MPI support")
     variant(
         "implicit_conversions",
-        default=True,
+        default=False,
         description="Enable implicit" "conversions to/from raw pointers",
     )
-    variant("benchmarks", default=True, description="Build benchmarks.")
-    variant("examples", default=True, description="Build examples.")
-    variant("docs", default=False, description="Build documentation")
     variant("tests", default=False, description="Build tests")
+    variant("benchmarks", default=False, description="Build benchmarks.")
+    variant("examples", default=False, description="Build examples.")
```

Comment:
I set those to "False" which is coherent with `tests` being set to "False" itself.
Especially true for `benchmarks` which requires `tests`.
@adayton1 Are you OK with that ?

Reply:
I'm OK with tests, benchmarks, examples, and docs all defaulting to False.